modules/ROOT/pages/transaction-management.adoc

```diff
@@ -147,6 +147,10 @@ When running in a transactional scope, the entire process must run in the same t
 
 Any other router or scope (for example, Foreach) continues with the transaction and its behavior is not modified.
 
+=== Execution Engine Selection
```

Comment:
should not be: "Execution Type Selection" ? because there is just one execution engine, but different execution types.